src/components/banner/banner.import.less

```diff
@@ -0,0 +1,74 @@
+.head-image {
+  width: 100%;
+  position: relative;
+  color: white;
+  font-size: 1.2em;
+  text-shadow: 6px 6px 0px rgba(0, 0, 0, 0.4);
+  background: repeating-linear-gradient(-55deg, #5200cc, #5200cc 10px, #4700b3 10px, #4700b3 20px);
+  padding-left: 25%;
+  padding-top: 50px;
+  padding-bottom: 50px;
+}
+.head-image .head-image-content {
+  width: 100%;
+}
+.head-image .head-image-content .home-image-inner-content {
```

Comment:
@urvi-pisal `.head-image-content` should be removed here. The other two classes provide enough style wrapping protection and specificity. 90% of your classes should be 2 deep, with the rest either being the root class (1 deep) or a hover state or pseudo element (3 deep)